Jenkinsfile

```diff
@@ -0,0 +1,357 @@
+// -*- mode: groovy -*-
```

Comment:
Do we have a pipeline in github? Does it make sense to include the jenkinsfile?

Reply:
We certainly don't have Jenkins set up. While it would be good to have some sort of pipeline in the future, for now I think we should just remove this file altogether.